src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((base * height / 2) * 10.0 / 10.0);
```

Comment:
All magic numbers in your code should be constants. The number 2 in the area calculation is a magic number and should be replaced with a named constant for clarity.